tests/check-airlock.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+# set DNS for RELEASE_HOST to 127.0.0.1
+# shellcheck source=/dev/null
+source /home/opensafely/config/load-env
+HOSTNAME="$(echo "$RELEASE_HOST" | cut -d'/' -f3 | cut -d':' -f1)"
+grep -q "$HOSTNAME" /etc/hosts || echo "127.0.0.1 $HOSTNAME" >> /etc/hosts
+
+# simple test of the running airlock
+curl -s --fail "$RELEASE_HOST" -o /tmp/airlock-homepage.html
+grep "<title>Airlock</title>" /tmp/airlock-homepage.html
```

Comment:
This could be brittle. I think a 200 is fine?

If we really want to, we could add a /healthcheck url?

Reply:
I agree, this is brittle, but I don't completely trust a 200 response - have been burnt before by things like tests getting 200 with completely wrong content. A `/healthcheck` url with some pre-defined content would be great?

Reply:
we can remove the content test if you think it's overkill though